### PR TITLE
Updating ruby-20 links and image name so they point to new repo and image

### PR DESCRIPTION
**Pull request by jhadvig**
This is a part of work on https://trello.com/c/UJG1mPZf/364-3-v3-ruby-image-image-beta2
Also last step before we cant get rid of the ruby-20-centos repository

### CODE REVIEW

**Comment by jhadvig** [comment]
@mfojtik @bparees PTAL

**Comment by mfojtik** [comment]
LGTM [test]

File: README.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ FAQ
     OpenShift is designed to run any existing Docker images.  In addition you can define builds that will produce new Docker images from a Dockerfile.  However the real magic of OpenShift can be seen when using [Source-To-Image](https://github.com/openshift/source-to-image)(STI) builds which allow you to simply supply an application source repository which will be combined with an existing STI-enabled Docker image to produce a new runnable image that runs your application.  We are continuing to grow the ecosystem of STI-enabled images and documenting them [here](https://ci.openshift.redhat.com/openshift-docs-master-testing/latest/openshift_sti_images/overview.html).  We also have a few more experimental images available:
 
     * [Wildfly](https://github.com/openshift/wildfly-8-centos)
-    * [Ruby](https://github.com/openshift/ruby-20-centos/)
+    * [Ruby](https://github.com/openshift/sti-ruby/)
```
Comment (bparees) [inline README.md]: Now that we have an official image, it should go into the openshift-docs and be removed from this experimental list.

File: examples/sample-app/application-template-stibuild.json [hunk 2]
```diff
@@ -111,8 +111,8 @@
         "strategy": {
           "type": "STI",
           "stiStrategy": {
-            "image": "openshift/ruby-20-centos",
-            "scripts": "https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin"
+            "image": "openshift/ruby-20-centos7",
+            "scripts": "https://raw.githubusercontent.com/openshift/sti-ruby/master/2.0/.sti/bin"
```
Comment (bparees) [inline examples/sample-app/application-template-stibuild.json]: we should strip scripts entirely, it's not needed.

File: README.md (hunk 1 above)
Comment (jhadvig) [inline README.md]: @bparees I've already send PR, which updates the the openshift-docs. You suggest to remove the ruby link ?

File: examples/sample-app/application-template-stibuild.json (hunk 2 above)
Comment (jhadvig) [inline examples/sample-app/application-template-stibuild.json]: @bparees you mean only here, or in all the BuillConfigs ?

File: README.md (hunk 1 above)
Comment (bparees) [inline README.md]: yeah, remove this link then.  it's only for our experimental images.

Comment (jhadvig) [inline README.md]: will :boom: it

File: examples/sample-app/application-template-stibuild.json (hunk 2 above)
Comment (bparees) [inline examples/sample-app/application-template-stibuild.json]: just here, i just don't want people copying our sample to think they need this.

**Comment by jhadvig** [comment]
@bparees updated

**Comment by bparees** [comment]
lgtm. [merge]

**Comment by mfojtik** [comment]
[merge]

**Comment by jhadvig** [comment]
[test]

**Comment by mfojtik** [comment]
[merge]

**Pull request merged**